src/data/repositories/consts/DiseaseOutbreakConstants.ts

```diff
@@ -33,3 +39,102 @@ export const DiseaseOutbreakCodes = {
     incidentManager: "RTSL_ZEB_TEA_ASSIGN_INCIDENT_MANAGER",
     notes: "RTSL_ZEB_TEA_NOTES",
 } as const;
+
+export type DiseaseOutbreakCode = GetValue<typeof DiseaseOutbreakCodes>;
+
+export type KeyCode = (typeof DiseaseOutbreakCodes)[keyof typeof DiseaseOutbreakCodes];
+
+export function isStringInDiseaseOutbreakCodes(code: string): code is KeyCode {
+    return Object.values(DiseaseOutbreakCodes).includes(code as KeyCode);
```

Comment:
[nit-picking] the orthodox way is casting the other way (which is safe): `(Object.values(DiseaseOutbreakCodes) as string[]).includes(code)`